package.json

```diff
@@ -59,6 +59,7 @@
     "@vitest/coverage-v8": "^2.1.4",
     "automd": "^0.3.12",
     "better-sqlite3": "^11.5.0",
+    "bun": "^1.1.34",
```

Comment:
`bun` being necessary for some tests wasn't listed in documentation, so I added it here for posterity. Feel free to include that `npm i -g bun` is necessary in documentation instead if we don't want it as a dependency.